File: GFG/Arrays/lrbyone.py
def leftrotate(arr):
    n=len(arr)
    temp=arr[0]
    for i in range(len(arr)-1):
        arr[i]=arr[i+1]
    arr[n-1]=temp
    return arr

brr=[1,2,3,4,5,6,7,8,9]
print(leftrotate(brr))

File: GFG/Arrays/lrbyone.cpp
#include<iostream>
#include<vector>
using namespace std;

void leftrot(vector<int> &arr){
    int temp=arr[0];
    int n= arr.size();
    for(int i=0;i<n-1;i++){
        arr[i]=arr[i+1];
    }
    arr[n-1]=temp;
}
int main (){
    vector<int> arr = {1,2,3,4,5,6,7,8,9};
    leftrot(arr);
    for(int i=0;i<arr.size();i++){
        cout<<arr[i]<<" ";
    }
    return 0;
}